docs/app.md

```diff
@@ -157,6 +158,40 @@ console.log(myChildApp.isRunning());
 console.log(myStopWithParentApp.isRunning());
 ```
 
+### App's `restartWithParent`
+
+Call `stop` then `start` on the child app when the parent app restarts.  Default value is `false`.
+It can also be defined as a function returning a boolean value.
+
+```js
+var myApp = new Marionette.Toolkit.App();
+
+var persistantChildApp = myApp.addChildApp('persistantChildApp', {
+  AppClass: Marionette.Toolkit.App,
+  restartWithParent: false
+});
+
+persistantChildApp.on('stop start', function(options) {
+    console.log(this.isRestarting());
+});
+
+// does not log
+myApp.restart();
+
+var restartingChildApp = myApp.addChildApp('restartingChildApp', {
+  AppClass: Marionette.Toolkit.App,
+  restartWithParent: true
+});
+
+restartingChildApp.on('stop start', function(options) {
+    console.log(this.isRestarting());
+});
+
+// logs true twice
```

Comment:
I'm not entirely convinced this is true for the code above.  I think prior to `restartingChildApp.on` you should start the app..  otherwise it hasn't started and stop won't be run I think.